docs/encyclopedia/nexus.mdx

```diff
@@ -0,0 +1,102 @@
+
+Nexus uses the Temporal queue-based Worker architecture to ensure reliable execution and automatic load balancing for Nexus Operations.
+When a caller Workflow schedules a Nexus Operation, Temporal atomically transfers the Nexus request to a Nexus state machine responsible for [at-least-once](/nexus/operations#execution-semantics) invocation of the Nexus Operation.
+
+Workers authenticate to thier Namespace with supported methods including mTLS client certificates or API Keys in Temporal Cloud.
+This establishes the caller's identity and caller namespace for Endpoint access control.
+Nexus handler Workers poll the Endpoint's target Task Queue for Nexus Tasks to process.
+
+Nexus connectivity across Namespaces is provided by the Temporal Platform. 
+Temporal Cloud connects all Namespaces within an Account and provides built-in Endpoint access controls.
+Self-hosted deployments support Nexus connectivity within a single Cluster and you can implement a custom Authorizer.
+
+![NexusYourCloud](/img/nexus/nexus-connect-queue-based-workers.png)
+
+When you execute a [Nexus Operation](/nexus/operations) in a caller Workflow, the caller and handler Workers interact with Temporal as follows:
+
+1. **RespondWorkflowTaskCompleted([ScheduleNexusOperation command](/references/commands#schedulenexusoperation))** \-
+is used by the caller Worker to schedule a Nexus Operation, which atomically hands off execution to the caller's Nexus Machinery.
+2. **PollNexusTaskQueue** \-
+is used by the handler Worker to recieve a [Nexus Task](/workers#nexus-task) to process, for example to start a Nexus Operation.
+3. **RespondNexusTaskCompleted** or **RespondNexusTaskFailed** \-
+is used by the handler Worker to return the Nexus Task result.
+[Asynchronous Nexus Operations](/nexus/operations) will return a Nexus Operation ID, that can be used to perform additional actions like canceling an Operation.
+[Synchronous Nexus Operations](/nexus/operations#synchronous-operation-lifecycle) will return the Operation result directly.
+The caller's Nexus Machinery recieves the result and records a [NexusOperation event](/references/events#nexusoperationstarted) in the caller's Workflow History.
+4. **PollWorkflowTaskQueue** \-
+is used by the caller Worker to recieve a Workflow Task with the [Nexus Operation event](/references/events#nexusoperationstarted) which may be Started, Completed, Failed, Canceled, or TimedOut.
+
+## Built-in Nexus Machinery
+
+The built-in [Nexus Machinery](/glossary#nexus-machinery) uses state-machine-based invocation and completion callbacks to guarantee at-least-once execution of Nexus Operations.
+It uses [Nexus RPC](/glossary#nexus-rpc) on the wire, a new protocol designed with Durable Execution in mind, to communicate across Namespaces within an Account.
+
+Nexus RPC supports arbitrary-duration Operations that extend beyond a traditional RPC — a key underpinning to connect durable executions within and across Namespaces, clusters, and regions.
```

Comment:
This can be merged into the previous section or you can at least omit the "to communicate across ..." part from it.

Reply:
good catch, fixed